### PR TITLE
Add statements for dependency installers

### PR DESCRIPTION
**Pull request by Stronkness**
Added statements for checking if files exists and versions of pip, npm and composer is installed. If so, install the dependencies from the files.

Currently, its ugly code. This is because i dont know the standard of how you want the code to be. It does the job. However, if required to be accepted i can try to make it more good looking.

As for Composer, i saw that you can also create a project from the composer file, I was not sure what exactly you wanted with this. So if im thinking wrong with Compose, please tell me and i will fix it.

Solves #5

### CODE REVIEW

File: hooks/post-clone.sh [hunk 1]
```diff
@@ -9,3 +9,34 @@ if [[ -d "$cwd/$folder" ]]; then
     cd "$cwd/$folder"
     $SHELL
 fi
+
+# Install dependencies if they exist, first checks if software is installed
+# npm, package.json
+if [[ -f "package.json" ]]; then
+  if which node &> /dev/null
```
Comment (Mte90) [inline hooks/post-clone.sh]: I should check for npm as it is installed separately from node

File: hooks/post-clone.sh [hunk 2]
```diff
@@ -9,3 +9,34 @@ if [[ -d "$cwd/$folder" ]]; then
...
+    else
+        echo "[REQUIREMENT] pip not installed. Can't install python dependencies"
+    fi
+fi
+
+# composer, composer.json
+if [[ -f "composer.json" ]]; then
+  if which composer &> /dev/null
+    then
+        composer.phar install
```
Comment (Mte90) [inline hooks/post-clone.sh]: the right command should be `composer install`

**Review by Mte90 — CHANGES_REQUESTED** [review]
With those 2 tiny changes the code is fine to get merged :-)

**Comment by Mte90** [comment]
Another thing that is possible to do to check the code style is using https://www.shellcheck.net/ but you are right it should be mentioned in the project's readme.

**Comment by Stronkness** [comment]
No bugs detected with shellcheck! :)

**Comment by Mte90** [comment]
Cool thanks for your contribute!

**Pull request merged**